osfoffline/polling_osf_manager/polling.py

```diff
@@ -585,7 +585,8 @@ def update_remote_file(self, local_file, remote_file):
             new_remote_file = yield from self.osf_query.upload_file(local_file)
         except FileNotFoundError:
             logger.warning(
-                'file not reuploaded on remote server because does not exist locally. inside create_remote_file_folder')
+                'file not reuploaded on remote server because does not exist locally. it probably downloaded incorrectly. inside create_remote_file_folder')
```

Comment:
Please add the name of the file to the warning if possible- the logs are more useful if we know which file is having the problem. (fileobj.name or fileobj.path)